App/src/Service/GroupExpenseBalancer.php

```diff
@@ -4,25 +4,28 @@
 
 use App\Entity\Bilan;
 use App\Entity\Expense;
+use Error;
 
 class GroupExpenseBalancer
 {
-    public function expenseBalancer($expenses)
+    /**
+     * @return array<string, Bilan>
+     */
+    public function expenseBalancer($expenses): array
     {
         /**
-         * @var Bilan[]
+         * @var array<string, Bilan>
          */
-        $bilans = array_reduce($expenses, static function (array $bilans, Expense $expense) {
-            $payer = $expense->getPayer();
-            if (!array_key_exists($payer, $bilans)) {
-                return [...$bilans, $payer => new Bilan($payer)];
-            }
-
-            return $bilans;
-        }, []);
+        $bilans = array_reduce(
+            $expenses,
+            static fn (array $bilans, Expense $expense) => array_key_exists($expense->getPayer(), $bilans)
+                ? $bilans
+                : [...$bilans, $expense->getPayer() => new Bilan($expense->getPayer())],
+            []
+        );
 
         $this->setExpenses($expenses, $bilans);
-
+        dump($bilans);
```

Comment:
nope 😄

Reply:
Yes, my bad I push to early i already have clean this...